client/my-sites/upgrades/navigation.jsx

```diff
@@ -30,6 +30,12 @@ var config = require( 'config' ),
 // remaining paths will make the button highlighted on that page.
 
 var NAV_ITEMS = {
+	Addons: {
```

Comment:
@scruffian: I think yes, for a couple of reasons: first of all there's the shopping cart off to the right of the cake and secondly, the nav item provides a heading of sorts to the page "Add-ons for Jetpack sites" which I'd like to be present.